### PR TITLE
Address several fixes from QA

### PR DESCRIPTION
**Pull request by Kelketek**
- [x] List current user in team list.
- [x] Upload message text not appearing until refresh
- [x] User grade display precision too high
- [x] Auto-select first team mate

### CODE REVIEW

**Comment by Kelketek** [comment]
@e-kolpakov This is ready for you.

File: group_project_v2/stage_components.py [hunk 1]
```diff
@@ -718,8 +722,12 @@ class ProjectTeamXBlock(
 
     def student_view(self, context):
         fragment = Fragment()
+        user_details = self.stage.project_api.get_user_details(self.stage.user_id)
+        user_organizations = self.stage.project_api.get_user_organizations(self.stage.user_id)
+        if user_organizations:
+            user_details.organization = user_organizations[0]['display_name']
```
Comment (e-kolpakov) [inline group_project_v2/stage_components.py]: @Kelketek duplicates existing [`team_member` property code](https://github.com/open-craft/xblock-group-project-v2/blob/master/group_project_v2/stage.py#L169-L172). Would be great to refactor it to something like

    def get_member_data(user_id):
        # this code

A good place to add it is the lower part of ProjectAPI class, where post-processing methods live (after the TODO "this methods post-process ....")

File: tests/unit/test_stage_components.py [hunk 2]
```diff
@@ -245,6 +249,9 @@ def test_upload_submission_success_scenario(self, submission_id, file_url, stage
             self.assertEqual(response_payload['title'], self.block.SUCCESSFUL_UPLOAD_TITLE)
             self.assertEqual(response_payload["submissions"], {submission_id: file_url})
             self.assertEqual(response_payload["new_stage_states"], [stage_state])
+            self.assertEqual(response_payload["user_label"], 'Test label')
+            # This isn't an especially strict regex, but if it passes it likely indicates this data is present and ok.
+            self.assertRegexpMatches(response_payload["submission_date"], r'[a-zA-Z]+ [0-9]+')
```
Comment (e-kolpakov) [inline tests/unit/test_stage_components.py]: @Kelketek `freezegun` + exact datetime matching.

**Comment by Kelketek** [comment]
@e-kolpakov Should be ready for a second look.

**Comment by e-kolpakov** [comment]
@Kelketek +1: than

**Pull request merged**